### ISSUE TITLE
App forgets login infos

### ISSUE THREAD

**Issue by gnppn** [gizmhail/iNoco#1]
Hi,

The iNoco app, as the AppStore comments say, forgets the login infos at each restart. It then asks to re-identify every time. Is it possible to correct it?

Thanks in advance,

GP.

**Comment by gizmhail** [comment]
Hi,

I assume you're french, so I'll switch to this language (as we would both be speaking it), but if I'm mistaken, don't hesitate to tell me and I'll translate.

La demande de login se fait à chaque démarrage ? Ou juste assez souvent.
Je demande, car on avait remarqué un souci qui pouvait faire assez régulièrement perdre l'authentification lié à l'extension Aujourd'hui (le widget) iNoco.

Dans le doute, si vous utilisez l'extension aujourd'hui/widget iNoco (soit dans le menu aujourd'hui, en swippant à gauche de l'accueil de l'iPhone, soit directement sur iNoco en faisant un force touch), est-ce que vous pourriez essayer de la désactiver, pour savoir si le souci vient bien de là, ou si c'est un nouveau problème ?

Si jamais le souci est bien celui lié à l'extension aujourd'hui iNoco, il faut que  je reteste, car il n'était pas aussi gênant avant (mais iOS 10, avec le mode widget, à peut-être rendu le déclenchement du souci plus courant...).
Si jamais ce n'est pas lié à l'extension, il faudra surement que je vous pose plus de question pour déterminer ce qui se passe (notamment, l'OS sur lequel vous tournez : iOS10, iOS9, ... et le modèle d'appareil : iPad, iPhone, ...).

Désolé du désagrément en tout cas.

**Comment by gnppn** [comment]
Oui, l'habitude de l'anglais ici :) Désolé pour le retard de la réponse.

Le problème se manifeste sur un iPhone avec le widget et un iPad sans. Les deux utilisent la dernière version de l'application sur iOS à jour (10.3.2) iPhone 6s/iPad mini 4.

N'hésitez pas si besoin de tests.
